### ISSUE TITLE
[DGIdGateway] not communicating with YSFGateway

### ISSUE THREAD

**Issue by dg9vh** [g4klx/YSFClients#253]
Hi,

as reported in the mailing list I have following problem: When trying to use YSFGateway (the original version from this repository) it is not replying on Wires-X activations. When pushing the Dx-Button on my FT1D, nothing happens, the hotspot receives my transmissions and as it is a dual_hat it puts it on the output, but no action follows from the YSFGatway.

Excpected behavior is to activate Wires-X-functionality and make the reflectorlist searchable with all-search or part-search functions of FT1D.

Maybe you can see the error here?

73 de Kim
DG9VH

**Comment by g4klx** [comment]
I'm not in a position to test anything here at the moment.
I would really like to kill off the YSF Gateway and move over to the DG-Id Gateway, it's more fun.
Remember Wires-X isn't documented so everything has been done in the dark. It would be good to get an up-to-date Wires-X dump from someone to see if anything has changed.
Jonathan  G4KLX


    On Sunday, 28 February 2021, 14:27:53 GMT, Kim - DG9VH <notifications@github.com> wrote:  
 
 


Hi,

as reported in the mailing list I have following problem: When trying to use YSFGateway (the original version from this repository) it is not replying on Wires-X activations. When pushing the Dx-Button on my FT1D, nothing happens, the hotspot receives my transmissions and as it is a dual_hat it puts it on the output, but no action follows from the YSFGatway.

Excpected behavior is to activate Wires-X-functionality and make the reflectorlist searchable with all-search or part-search functions of FT1D.

Maybe you can see the error here?

73 de Kim
DG9VH

**Comment by dg9vh** [comment]
Yes, you are primary right... in DGIdGateway we have the possibility to set reflectors static or "on demand" so we can put up to 99 reflectors into the list that can be used... I have a bunch of 5 reflectors listed static here... so under this point of view we do not need YSFGateway :-)

Should we stay open here or close it? It was only an observation I made and nothing that kills me in working :-) So we can leave it open for later or close it.

73 Kim

**Comment by g4klx** [comment]
It would be nice if someone else were able to look into it. I can't be the only person with C++ skills!

**Comment by K9bdh** [comment]
So the DGID gateway does work?   We have 3 fusion repeaters linked together via IMRS.  So would i need to go into. DGID gateway and put the IP addresses of each repeater in the set up file.    All 3 repeaters are linked together on DGID 01
